feature-pack/src/license/full-feature-pack-licenses.xml

```diff
@@ -1991,6 +2057,20 @@
         </license>
       </licenses>
     </dependency>
+    <dependency>
+      <groupId>org.hdrhistogram</groupId>
+      <artifactId>HdrHistogram</artifactId>
+      <licenses>
+        <license>
+          <name>Public Domain, per Creative Commons CC0</name>
+          <url>http://creativecommons.org/publicdomain/zero/1.0/</url>
+        </license>
+        <license>
+          <name>BSD-2-Clause</name>
+          <url>https://opensource.org/licenses/BSD-2-Clause</url>
+        </license>
+      </licenses>
+    </dependency>
```

Comment:
These two are not quite correct, but I'll file an issue to fix these later as it's not important until a release is to be done. Seems like something that should not hold up the PR.

Reply:
Just linking the JIRA here https://issues.redhat.com/browse/WFLY-12909

Reply:
WFLY-12909 is resolved now, resoving here as well.